src/app/components/passphrase-form/passphrase-form.component.ts

```diff
@@ -6,32 +6,64 @@ import { WalletService } from "src/app/services/wallet.service";
 @Component({
   selector: "app-passphrase-form",
   templateUrl: "./passphrase-form.component.html",
-  styleUrls: ["./passphrase-form.component.scss"]
+  styleUrls: ["./passphrase-form.component.scss"],
 })
 export class PassphraseFormComponent implements OnInit {
   @Output() cancelled = new EventEmitter<void>();
   @Output() created = new EventEmitter<string>();
   duplicatedWalletId = false;
-  walletId: string;
+  currentWalletId: string;
+  walletPassphrase: string = "123";
+  walletEnckey: string = "abc";
```

Comment:
Is these default value still used here? =P

Reply:
it's not used any more, i'll remove it